openapi/spec.yaml

```diff
@@ -9,6 +9,43 @@ servers:
     description: localhost
 
 paths:
+  /otpcount/{operationType}:
+    get:
```

Comment:
don't check-in the changes to the spec, if you want keep those as your locak copy.